server/routes/induction/create/reasonsNotToGetWorkCreateController.test.ts

```diff
@@ -0,0 +1,202 @@
+import { Request, Response } from 'express'
+import type { SessionData } from 'express-session'
+import type { InductionDto } from 'inductionDto'
+import type { ReasonsNotToGetWorkForm } from 'inductionForms'
+import aValidPrisonerSummary from '../../../testsupport/prisonerSummaryTestDataBuilder'
+import ReasonsNotToGetWorkCreateController from './reasonsNotToGetWorkCreateController'
+import ReasonNotToGetWorkValue from '../../../enums/reasonNotToGetWorkValue'
+import QualificationLevelValue from '../../../enums/qualificationLevelValue'
+
+describe('reasonsNotToGetWorkCreateController', () => {
+  const controller = new ReasonsNotToGetWorkCreateController()
+
+  const prisonNumber = 'A1234BC'
+
+  let req: Request
+  let res: Response
+  const next = jest.fn()
+
+  const noErrors: Array<Record<string, string>> = []
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    req = {
+      session: {} as SessionData,
+      body: {},
+      user: {} as Express.User,
+      params: { prisonNumber } as Record<string, string>,
+      flash: jest.fn(),
+    } as unknown as Request
```

Comment:
I think you're going to need to set the path property in the mock request for "this" page